modules/dnn/src/ie_ngraph.cpp

```diff
@@ -401,9 +406,9 @@ InfEngineNgraphNode::InfEngineNgraphNode(const std::vector<Ptr<BackendNode> >& n
     : BackendNode(DNN_BACKEND_INFERENCE_ENGINE_NGRAPH), cvLayer(cvLayer_)
 {
 #if INF_ENGINE_VER_MAJOR_GE(INF_ENGINE_RELEASE_2020_3)
-    ngraph::OutputVector inp_nodes;
+    ov::OutputVector inp_nodes;
 #else
-    ngraph::NodeVector inp_nodes;
+    ov::NodeVector inp_nodes;
```

Comment:
Is `ov::` available for `< 2020.3`?

Reply:
do you need to support 4 years old OpenVINO?
Can we say only 2022.1 and newer are supported? It will significantly simplify conditions in code where OpenVINO is used and also will allow to drop legacy cmake code and rely only on `find_package(OpenVINO)`

Reply:
Agree. For now CI works on 2021.4 so we can keep it to test it against OpenVINO API 2.0 (2022.x). I just mean that this lane changed under `#ifdef` and we don't test it. So specifically for this line, I recommend revert. Let's drop OpenVINO support for 2020.x in a separate PR.

Reply:
>  2021.4

API 2.0 since 2022.1
If we want to drop IE:: and API 1.0, we need to test since 2022.1

Reply:
@dkurt are you OK to drop releases older 2022.1 ? to keep API 2.0 only

Reply:
I don't mind. Created an issue here: https://github.com/opencv/opencv/issues/25207.

This way https://pullrequest.opencv.org/buildbot/builders/4_x_openvino-opencl-skl-lin64 should be moved to 2022.1 or newer (/cc @asmorkalov, @opencv-alalek)

I only recommend to do this in a separate PR

Reply:
> I only recommend to do this in a separate PR

But in this case we need to do double job:
- Current PR supports both Pre and Post API 1.0 changes
- Next PR removes Pre API 1.0 code path.

I suppose we need to do it in one shot and just drop API 1.0 here

Reply:
Sure, @olpipi, will you handle OpenVINO API 1.0 removal? Please keep 2022.1 is a minimal supported version.